src/main/java/com/cs56fitnessapp/models/User.java

```diff
@@ -7,5 +7,16 @@
  */
 public class User {
 
-
+    private User  user;
```

Comment:
User is underlined in the UML - meaning static. Should be `private static User`. 
Remove extra space after the word `User`.
Do so for every field where there 2 spaces (instead of one) after the type declaration.